packages/block-library/src/footnotes/style.scss

```diff
@@ -19,3 +19,9 @@ a[data-fn].fn::after {
 	text-indent: 0;
 	float: left;
 }
+
+sup[data-fn].fn {
```

Comment:
At the very least, this should be at zero specificity (wrapped in `:where()`) since these are reset styles?